internal/bminventory/inventory.go

```diff
@@ -3481,7 +3487,17 @@ func (b *bareMetalInventory) GetClusterInternal(ctx context.Context, params inst
 		host.FreeAddresses = ""
 	}
 
-	infraEnv, err := common.GetInfraEnvFromDB(b.db, *cluster.ID)
+	imageInfo, err := b.getImageInfo(cluster.ID)
```

Comment:
I think that it should be infra-env, in v2 api cluster id and infra env id are not the same, @yevgeny-shnaidman please correct me if i'm wrong.